ITR/data/base_providers.py

```diff
@@ -36,6 +36,9 @@ def __init__(self, production_benchmarks: IProductionBenchmarkScopes,
         self.temp_config = tempscore_config
         self.column_config = column_config
         self._productions_benchmarks = production_benchmarks
+        # production benchmarks are input for S1S2, but relevant for all scopes
+        self._productions_benchmarks.S3 = self._productions_benchmarks.S1S2
```

Comment:
My previous comment is an example of the kind of confusion we have in the code about production benchmarks being conditioned on SCOPE.  The fact is, production is produced (t Steel, GWh) regardless of emissions.  It makes no sense to carry around 5 different copies of the same production data (one for each of S1, S2, S1S2, S3, S1S2S3).  Better to clean all that up and make productions free of scope.  It's easy to edit the JSON files to correct.

Reply:
Agree, I will make production free of scope

Reply:
all right, i pushed the change. Production benchmark now comes with a common field "AllScopes"